doc/classes/Quaternion.xml

```diff
@@ -118,6 +118,30 @@
 				The order of each consecutive rotation can be changed with [param order] (see [enum EulerOrder] constants). By default, the YXZ convention is used ([constant EULER_ORDER_YXZ]): Z (roll) is calculated first, then X (pitch), and lastly Y (yaw). When using the opposite method [method from_euler], this order is reversed.
 			</description>
 		</method>
+		<method name="get_rotation_around" qualifiers="const">
+			<return type="Quaternion" />
+			<param index="0" name="axis" type="Vector3" />
+			<description>
+				Returns the quaternion's rotation [b]around[/b] the normalized axis vector. The quaternion and the axis vector must be normalized, otherwise it returns the identity quaternion.
+				This method is related to the swing-twist decomposition, but provide some safe-check and a proper name.
```

Comment:
```suggestion
				This method uses swing-twist decomposition (see [method get_swing] and [method get_twist]), but provides some validation and a clear name.
```
Not 100% sure "uses" is better or even correct here, so this is a suggestion, not a requiement; "is related to" just seemed a bit vague.